src/axom/sidre/tests/sidre_buffer.cpp

```diff
@@ -442,7 +442,8 @@ TEST_P(UmpireTest, reallocate)
 {
   constexpr int SIZE = 100;
 
-  #if defined(AXOM_USE_CUDA) && defined(UMPIRE_ENABLE_CONST)
+  #if(defined(AXOM_USE_CUDA) || defined(AXOM_USE_HIP)) && \
```

Comment:
Should we define an `AXOM_USE_DEVICE` which is set when either `AXOM_USE_CUDA` or `AXOM_USE_HIP`?

Reply:
Update: Looks like we have `AXOM_USE_GPU`.
Please use this consistently throughout.

See:
https://github.com/LLNL/axom/blob/223142718829f9590f652684a8ccb294fdfaf567/src/axom/core/Macros.hpp#L77-L84